packages/esm-patient-allergies-app/src/allergies/allergies-form/allergy-form.component.tsx

```diff
@@ -154,7 +166,7 @@ function AllergyForm({ closeWorkspace, patientUuid }: DefaultWorkspaceProps) {
           (response: FetchResponse) => {
             if (response.status === 201) {
               mutate();
-              closeWorkspace();
+              closeWorkspace({ ignoreChanges: true });
```

Comment:
Huh, I see. Now that `ignoreChanges` is false by default, we have to pass this argument on saving & exiting, which is a little awkward. Nevertheless I still think this is the right approach.

Reply:
Actually, I guess the thing that is weird is actually that `isDirty` is still true after the form is submitted. Logically, it seems like it shouldn't be.